src/compiler/types.ts

```diff
@@ -3857,12 +3857,7 @@ namespace ts {
     export interface ResolvedModule {
         /** Path of the file the module was resolved to. */
         resolvedFileName: string;
-        /**

```

Comment:
This comment didn't describe what we were actually doing. We don't validate the contents of files in any way while doing module resolution.